CONTRIBUTING.md

```diff
@@ -9,13 +9,17 @@
       - [Description](#description)
       - [Level](#level)
       - [Author](#author)
-    - [Optional field best practices](#optional-field-best-practices)
       - [Summary](#summary)
+    - [Optional field best practices](#optional-field-best-practices)
+      - [Title](#title)
+      - [PackUrl](#packurl)
```

Comment:
Don't think this belongs in the contributing guide (see other comment as well in the config-spec.md file)

Reply:
Removed.